src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -169,30 +169,41 @@ uiModules.get('apps/management')
           notify.error('The file could not be processed.');
         }
 
-        return Promise.map(docs, (doc) => {
-          const { service } = find($scope.services, { type: doc._type }) || {};
-
-          if (!service) {
-            const msg = `Skipped import of "${doc._source.title}" (${doc._id})`;
-            const reason = `Invalid type: "${doc._type}"`;
-
-            notify.warning(`${msg}, ${reason}`, {
-              lifetime: 0,
-            });
-
-            return;
-          }
-
-          return service.get().then(function (obj) {
-            obj.id = doc._id;
-            return obj.applyESResp(doc).then(function () {
-              return obj.save({ confirmOverwrite : true });
-            });
-          });
+        return new Promise((resolve) => {
+          confirmModal(
+            `Would you like to overwrite any existing saved objects?`, {
```

Comment:
I asked @debadair to help us here 😄 

Here's the dialog, for quick reference:

<img width="487" alt="screen shot 2017-03-03 at 2 07 02 pm" src="https://cloud.githubusercontent.com/assets/51061/23570792/c3107c16-001a-11e7-9447-3a2d3ccf04e4.png">